Makefile

```diff
@@ -79,6 +75,42 @@ update-requirements: ## Updates requirement
 #	echo "-e git://github.com/wellcometrust/WellcomeML.git@tokenizer-decode#egg=wellcomeml" >> requirements.txt
 	echo "git+https://github.com/nsorros/shap.git@dev" >> requirements.txt
 
+
+virtualenv-dev: ## Creates virtualenv
+	@if [ -d $(VIRTUALENV) ]; then rm -rf $(VIRTUALENV); fi
+	@mkdir -p $(VIRTUALENV)
+	virtualenv --python $(PYTHON) $(VIRTUALENV)
+	$(PIP) install --upgrade pip
+	$(PIP) install pytest pytest-cov tox
+	$(PIP) install -r dev_requirements.txt
+	$(PIP) install --no-deps -e .[dev]
+	$(PIP) install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_core_sci_sm-0.4.0.tar.gz
+	$(VIRTUALENV)/bin/pre-commit install --hook-type pre-push --hook-type post-checkout --hook-type pre-commit
+
+update-requirements: VIRTUALENV := /tmp/update-requirements-venv/
```

Comment:
this is a duplicate and it creates a problem when run `make update-requirements`
```
Makefile:92: warning: overriding commands for target `update-requirements'
Makefile:68: warning: ignoring old commands for target `update-requirements'
```

so keep only one of them

Reply:
removed duplicate